### ISSUE TITLE
Implement Zoomable video

### ISSUE THREAD

**Issue by puregallery** [puregallery/Pure-Gallery-Support#30]

**Comment by Nostang3** [comment]
PG exoplayer now allows for zoom. Tested working.

**Comment by puregallery** [comment]
Pending:
1. Add an option to chose the default player.
2. Add the option to settings page.

**Comment by Nostang3** [comment]
I was going to mention the default options. This also reminded me to add some issues. Now I just need to remember what they were. Lol.

**Comment by puregallery** [comment]
No worries...I do have a couple of outstanding issues and that is the reason I marked that feature beta(in a beta app, smh)...
Anyway, the issues:
 1. Can't hide the controls properly.
 2. While zooming, the view doesn't resize the height properly and as a result, the focus point gets lost. 

Turns out zooming a video is wayy different than zooming an image. I have a few ideas regarding this and need to test them out.
